### ISSUE TITLE
Worse Icons and no forecasting

### ISSUE THREAD

**Issue by veratechnz** [apilayer/weatherstack#2]
You just lost a potential customer. Your new api has degraded icons and no forecasting on your free plan. When my app goes commercial I was planning to upgrade to a paid plan, but your new api and it's degraded service is a step backwards. #worseicons #apixuwascooler.

**Comment by gkelley** [comment]
Could we revert to the old icons?

**Comment by EdwardD20** [comment]
I agree with @veratechnz, one forecast day would be better than none on the free plan. Can't even figure out how to code for forecasts. I'm writing in Swift and I can't figure out how to code for a dynamic value (date) from the start. Historical would be the same issue if I wanted to support that.

As for @gkelley, manually code your own icons (use previous ones)?

**Comment by jerzeemayana** [comment]
@veratechnz sure u re right bro.........

**Comment by vesakk** [comment]
Any news???

**Comment by EdwardD20** [comment]
Sorry. The official Meteorologist channel is on SourceForce.net ([https://sourceforge.net/p/heat-meteo/bugs/](https://sourceforge.net/p/heat-meteo/bugs/)). I have not been watching this thread but did get notified so I guess it works anyway.

The latest version of Meteorologist, 3.4.2, was released in July 2020. Not sure what version you are currently running. You can choose between both the old and new icons so that is not an issue. As for limitations of ApiLayer/WeatherStack, that's up to them to provide options. Meteorologist is a free app that consumes the data they provide.

Ed

https://sourceforge.net/projects/heat-meteo/

**Comment by EdwardD20** [comment]
Sorry everybody, my mind got crossed with regard to this thread. Meteorologist uses WeatherStack as a weather source and I got confused who was asking what.

Ed
